src/Data/List/Relation/Binary/Permutation/Setoid/Properties.agda

```diff
@@ -29,19 +30,20 @@ import Data.List.Relation.Unary.Unique.Setoid as Unique
 import Data.List.Membership.Setoid as Membership
 open import Data.List.Membership.Setoid.Properties using (∈-∃++; ∈-insert)
 import Data.List.Properties as List
-open import Data.Nat.Base using (ℕ; suc; _<_; z<s; _+_)
+open import Data.Nat.Base using (ℕ; suc; 2+; _<_; z<s; _+_)
```

Comment:
I'm irritated that we didn't make this pattern synonym `2+_`... leading to the mistake in my suggestion here. But should be good now!